workloads/spec-reference.md

```diff
@@ -0,0 +1,1095 @@
+any images pushed under that name. This makes the content-addressable name, such as
+the image name containing the digest, available for further resources in the
+supply chain.
+
+Parameters:
+
+<table>
+  <tr>
+    <th>parameter name</th>
+    <th>meaning</th>
+    <th>example</th>
+  </tr>
+
+  <tr>
+    <td><code>serviceAccount</code></td>
+    <td>
+      name of the serviceaccount (in the same namespace as the Workload) to use
+      for providing the necessary credentials to `ImageRepository` for fetching
+      the container images.
+    </td>
+    <td>
+      <pre>
+      - name: serviceAccount
+        value: default
+      </pre>
+    </td>
+  </tr>
+</table>
+
+> **Note** `--service-account` flag sets the `spec.serviceAccountName` key in
+> the Workload object. To configure the `serviceAccount` parameter, use
+> `--param serviceAccount=...`.
+
+For information about the
+ImageRepository resource, see [ImageRepository reference
+docs](../source-controller/reference.hbs.md#imagerepository).
+For information about the prebuild image function, see [Using a prebuilt
+image](pre-built-image.hbs.md).
+
+## image-scanner
+
+The `image-scanner` resource is included only in
+`ootb-supply-chain-testing-scanning`. This resource is responsible for scanning a container image, either built by using
+the supply chain or prebuilt,
+persisting the results in the store, and gating the image from moving
+forward in case the CVEs found are not compliant with the ScanPolicy referenced
+by the ImageScan object create for doing so.
+
+Parameters:
+
+<table>
+  <tr>
+    <th>parameter name</th>
+    <th>meaning</th>
+    <th>example</th>
+  </tr>
+
+  <tr>
+    <td><code>scanning_image_template</code></td>
+    <td>
+      Name of the ScanTemplate object in the same namespace as the Workload to
+      make use of for running the scans against a container image.
+    </td>
+    <td>
+      <pre>
+      - name: scanning_image_template
+        value: private-image-scan-template
+      </pre>
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>scanning_image_policy</code></td>
+    <td>
+      Name of the ScanPolicy object in the same namespace as the Workload to
+      make use of when evaluating the scan results of an image scan.
+    </td>
+    <td>
+      <pre>
+      - name: scanning_image_policy
+        value: allowlist-policy
+      </pre>
+    </td>
+  </tr>
+</table>
+
+For information about the ImageScan custom resource, see [ImageScan reference](../scst-scan/scan-crs.hbs.md#imagescan).
+
+For information about how the artifacts found
+during scanning are catalogued, see [Supply Chain Security Tools for Tanzu –
+Store](../scst-store/overview.hbs.md).
+
+## config-provider
+
+The `config-provider` resource in the supply chains is responsible for
+generating a PodTemplateSpec
+to use in app configs, such as knative services and deployments, to
+represent the shape of the pods that you want to instantiate to run the
```

Comment:
It might be a bit of a stretch. How about we go with something like:

```suggestion
to use in app configs, such as knative services and deployments, which
represent the desired pod configuration to instantiate to run the
```